.changeset/solid-cloths-cheat.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/core': patch
+---
+
+Fixed runEvals to preserve failed items with trace links and mark tool validation errors as unsuccessful trajectory steps.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)